lirc_client.py

```diff
@@ -227,41 +231,26 @@ class AbstractLircClient:
     _last_remote = None
 
     def __init__(self, verbose, timeout):
-        self._verbose = verbose
-        self._timeout = timeout
+        self.timeout = timeout
+        self.verbose = verbose
         self._parser = ReplyParser()
-        self._in_buffer = ''
+        self._in_buffer = bytearray(0)
         self._last_command = None
 
-    def set_verbosity(self, verbosity):
-        """Set verbosity to the value of the argument."""
-        self._verbose = verbosity
-
```

Comment:
You are missing the point here. It is _NOT_ good OO programming to slam in a getter and a setter to every private variable, and it was not done that way either. Note that there were no getter (i.e. no read access) to `verbosity` and `timeout`. There may possibly be argument for allowing read access, but "it is more Pythonic" is surely not a convincing argument for loosening the access control.